### PR TITLE
Demangle Cloudflare's email mangling.

### PR DESCRIPTION
**Pull request by jmgao**

### CODE REVIEW

**Comment by jmgao** [comment]
I started reading Ward with this (thanks!), and this bugged me enough to fix it.
[Total diff of ward before and after](https://gist.github.com/jmgao/d1963f57ea95410aa2ee279a5a713b85)

File: Cargo.toml [hunk 1]
```diff
@@ -10,3 +10,6 @@ gen-epub-book = "2.3.2"
 chrono = "0.4"
 url = "1.7.0"
 argparse = "0.2.2"
+
+regex = "1.3.7"
+hex = "0.4.2"
```
Comment (nicohman) [inline Cargo.toml:15]: What is this dependency being added for? I don't see it being used anywhere.

Comment (nicohman) [reply Cargo.toml:15]: NVM I see it

**Comment by nicohman** [comment]
Looks good. Thanks!

**Pull request merged**